rozwiazanie 1

iordakowska opened the pull request:
Obecnie potrawy według kategorii wyświetlają się na stronie głównej, ale chciałabym aby wyświetlały się na stronie allCategory.html. Niestety nie wiem jak to zrobić, żeby przycisk był na stronie głównej a przekierowanie do /allCategory/?category=DINNER. 
Cały czas mam problem z GetMappingiem
Na home.html :
<a class="btn btn-primary" href="/allCategory/?category=CAKE" role="button">Ciasta</a>
ale niestety 404 się poajwia

iordakowska committed:
rozwiazanie 1

slawekludwiczak reviewed (commented):
Film z podpowiedziami https://youtu.be/B6eg0gjHJ6s

iordakowska committed:
rozwiazanie 2
poprawa
dodawanie przepisu

slawekludwiczak commented:
ciąg dalszy sugestii https://youtu.be/oileXhAVinM

iordakowska committed:
rozwiazanie 3
problem z edycją przepisu- komentarz w recipeEdit.html
Proszę o pomoc

slawekludwiczak commented:
https://youtu.be/XZB2n3mnJ2A

iordakowska committed:
rozwiazanie 4
Edycja , usuwanie jest
Polubienia - wyrzuca błąd więc póki co zakomentowane
Nie mogę ustalić metody w recipeRepository, która by mi sortowała findAllByLikeDesc() -rzuca wyjątek

slawekludwiczak commented:
Na filmie pokazane jak poradzić sobie z polubieniami. Bez relacji (drugiej klasy encji) za to zadanie zdobędziesz maksymalnie 9 pkt
https://youtu.be/o0kms-oatRQ

iordakowska committed:
rozwiazanie 4
polubienia i sortowanie

slawekludwiczak reviewed (commented):
To co jeszcze jest do poprawy / dodania:
- poprawki kosmetyczne (wyrównanie przycisków na stronie głównej, różne nagłówki w zależności od tego, czy jesteśmy na stronie wszystkich przepisów, czy najlepszych przepisów, dodanie jakiejś grafiki na stronie z listami przepisów) (1 pkt)
- przerobienie Category z enuma na klasę tak, aby między Recipe i Category była relacja na poziomie bazy danych (3 pkt)
https://youtu.be/A392cTSHZ8Q

iordakowska committed:
rozwiazanie 5
2 bazy Category i Recipe
Problem:
Dodawanie, edycja
Nie wyświetla się obrazek na stronie kategorii
Proszę o pomoc

iordakowska committed:
rozwiazanie 6
Wszystko poprawione według wskazówek
2 encje
8 podpunktów
checkstyle
tylko image nie wyświetla się poprawnie- nie wiem dlaczego
